pkg/storage/minio/all_test.go

```diff
@@ -15,11 +16,19 @@ type MinioTests struct {
 }
 
 func (d *MinioTests) SetupTest() {
+	// TODO: what is the difference between all_test and test_suite.go??
```

Comment:
@rchakra3 I don't see a `test_suite.go` - do you know where that is?

Reply:
it's in the same directory - but you can ignore this! That was a comment for myself as I was reading through code - I realized later the `pkg/storage/*/test_suite.go` files implement the storage test suite interface used in `pkg/storage/storage_tests/`